data/json/npcs/TALK_FACTION_CAMP.json

```
@@ -34,9 +34,31 @@
   {
     "id": "TALK_CAMP_TUTORIAL_INTRO",
     "type": "talk_topic",
-    "dynamic_line": "The faction camp system is designed to give you greater control over your companions by allowing you to assign them to their own missions.  These missions can range from gathering and crafting to eventual combat patrols.",
+    "dynamic_line": "The faction camp system allows you to use friendly NPCs to take over existing buildings or construct new buildings and use them to extend the usefulness of your allies.  When you have a faction camp, you can order your companions to perform missions, such as crafting new items, building up the camp, hunting for food, or recruiting more allies.  Faction camps also simplify some aspects of managing your companions.",
     "responses": [
-      { "text": "Go on.", "topic": "TALK_CAMP_TUTORIAL_FOOD" },
+      { "text": "Give me an overview of how camps work.", "topic": "TALK_CAMP_TUTORIAL_OVERVIEW" },
+      { "text": "Tell me more about faction camps and food.", "topic": "TALK_CAMP_TUTORIAL_FOOD" },
+      { "text": "Tell me more about camp missions.", "topic": "TALK_CAMP_TUTORIAL_MISSIONS" },
+      { "text": "Tell me about building a camp.", "topic": "TALK_CAMP_TUTORIAL_BUILDING" },
+      { "text": "Give me some advice on building a camp.", "topic": "TALK_CAMP_TUTORIAL_ADVICE" },
+      { "text": "I know the basics.  Just tell me what changed.", "topic": "TALK_CAMP_TUTORIAL_CHANGES" },
+      { "text": "Never mind, let's go back to talking about camps.", "topic": "TALK_CAMP_GENERAL" },
+      { "text": "Never mind, let's talk about other things you can do.", "topic": "TALK_ALLY_TUTORIAL" },
+      { "text": "Never mind, let's talk about something else.", "topic": "TALK_FRIEND" },
+      { "text": "Forget it.  Let's go.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_CAMP_TUTORIAL_OVERVIEW",
+    "type": "talk_topic",
+    "dynamic_line": "Each faction camp starts with a bulletin board.  You can 'e'xamine the bulletin board to get a list of available missions with descriptions, and to assign an NPC to work on a mission.  The bulletin board is the primary way to access the camp.\n  If both you and a companion have two-way radios, you can assign missions to the companion remotely by talking on the radio.\n  When you ask a companion to start a camp, they'll build a bulletin board at the current location, and that will establish a faction camp at that overmap tile.  A faction camp has a central tile where the bulletin board is, and possibly one or more expansions that are in adjacent tiles.\n  All of your faction camps can be stocked with food, and your companions will eat from that food when performing camp missions, or even when they're just hungry and near the camp.\n  Currently, faction camps can be created in fields, in firestations, and in some evac shelters.",
```

firestations -> fire stations
```suggestion
    "dynamic_line": "Each faction camp starts with a bulletin board.  You can 'e'xamine the bulletin board to get a list of available missions with descriptions, and to assign an NPC to work on a mission.  The bulletin board is the primary way to access the camp.\n  If both you and a companion have two-way radios, you can assign missions to the companion remotely by talking on the radio.\n  When you ask a companion to start a camp, they'll build a bulletin board at the current location, and that will establish a faction camp at that overmap tile.  A faction camp has a central tile where the bulletin board is, and possibly one or more expansions that are in adjacent tiles.\n  All of your faction camps can be stocked with food, and your companions will eat from that food when performing camp missions, or even when they're just hungry and near the camp.\n  Currently, faction camps can be created in fields, in fire stations, and in some evac shelters.",
```

Also could you clarify "_some_ evac shelters" please?